js/script.js

```diff
@@ -67,6 +67,17 @@ const createTaskCard = task => {
     return taskCard;
 };
 
+const isMatchingId = (task, taskId) => {
+    return task.id === taskId;
+};
```

Comment:
```suggestion
const isMatchingId = (task, taskId) =>  task.id === taskId;
```

Reply:
Hello @aka-wtag 
Thanks for you review here.
However please understand that, being verbose is never bad specially if you are going to maintain and support the project for long time. 

Also this type of scenarios where either one works, but as a reviewer you "prefer" one way or another is often refrained upon. 

Thanks